src/subscriptions-client.js

```diff
@@ -614,7 +614,7 @@ client.getSyspurpose = function() {
                 })
                 .catch(ex => {
                     console.debug(ex);
-                    client.syspurposeStatus.info = 'Unknown'; // TODO: change to something meaningful
+                    client.syspurposeStatus.info = _('Unknown'); // TODO: change to something meaningful
```

Comment:
hmm is this ever used, even in untranslated form?

From what I see, this is passed to SubscriptionView:
https://github.com/candlepin/subscription-manager-cockpit/blob/c99923665afdb7904a7e89bda82f357000a7c079/src/index.js#L133-L134
then it is passed to SubscriptionStatus:
https://github.com/candlepin/subscription-manager-cockpit/blob/c99923665afdb7904a7e89bda82f357000a7c079/src/subscriptions-view.jsx#L543
and then in `SubscriptionStatus.render()` it is checked as hash for elements:
https://github.com/candlepin/subscription-manager-cockpit/blob/c99923665afdb7904a7e89bda82f357000a7c079/src/subscriptions-view.jsx#L334-L335
So IMHO
- the value here assigned to `info` should be `{}`  (or `null` after fixing `SubscriptionStatus.render()` to account for that)
- the fallback value (set in `catch()`) of the above `getSyspurposeStatus()` should be a translated string, as that is shown as-is

How did you get in this specific situation?

Reply:
good point @ptoscano ! Fixed it

> How did you get in this specific situation?

I saw screenshot and assumed it was this - apparently I was wrong.

> the fallback value (set in catch()) of the above getSyspurposeStatus() should be a translated string, as that is shown as-is

Right, it is `null` by default which shows empty fields on the page. I haven't touched this as you might want to distinguish between loading , loaded and failed states. I will leave this one to someone who knows this plugin better :)